### ISSUE TITLE
Not showing up when being removed

### ISSUE THREAD

**Issue by MicMicMon** [joeattardi/emoji-button#18]
There is a delay between "closing" the emoji picker and its `div` being removed from the DOM (for the nice fade out animation effect).

If `showPicker` is called while this animation takes place, the picker does not show up. I guess it's because the `div` is removed from the DOM a few milliseconds after.

As far as I understand, a check on whether the `visible` class is present on the DOM in `showPicker` could fix the issue.

Or maybe adapting the code I use to call the picker?
```typescript
this.picker.pickerVisible
      ? this.picker.hidePicker()
      : this.picker.showPicker(this.emojiButton.nativeElement);
```

**Comment by joeattardi** [comment]
I just released v2.6.1 which should fix this issue. Let me know if you are still having problems!

**Comment by MicMicMon** [comment]
Awesome, I confirm it fixes the issue! Thanks
